flight/Modules/Actuator/actuator.c

```diff
@@ -480,6 +481,10 @@ static void post_process_scale_and_commit(float *motor_vect,
 	 */
 	float maxpoweradd = (*maxpoweradd_bucket) / hangtime_leakybucket_timeconstant;
 
+	if (flip_over_mode) {
```

Comment:
The core changes are down here in actuator to A) don't do any kind of clip management, and B) when clipping, ensure the motors do not spin, and C) to do some thresholding / deadbanding on axes before applying the mixer.  Stabilization selects whether it wants this behavior, and passes the information down.  Actuator acts upon it.